### PR TITLE
Upgrade k8s version from 1.19.3 to 1.19.4 in upgrade workflow.

### PR DESCRIPTION
**Pull request by smoshiur1237**
Previously the upgrade of k8s version was done from 1.18.8 to 1.8.9. But at this moment we are using 1.19.3 for metal3 deployment. So to avoid the conflict in upgrade workflow, we upgrade k8s version from 1.19.3 to 1.19.4.

### CODE REVIEW

**Comment by smoshiur1237** [comment]
/assign @Xenwar 
/cc @kashifest 
//test-upgrade-features

File: scripts/feature_tests/upgrade/controlplane_upgrade/3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ if [[ "${FROM_VERSION}" < "${UPGRADED_K8S_VERSION_2}" ]]; then
 elif [[ "${FROM_VERSION}" > "${KUBERNETES_VERSION}" ]]; then
   TO_VERSION="${KUBERNETES_VERSION}"
 else
-  exit 0
+  exit 1
```
Comment (Xenwar) [inline scripts/feature_tests/upgrade/controlplane_upgrade/3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh:59]: add an echo with the proper description

**Review by Xenwar — CHANGES_REQUESTED** [review]

File: scripts/feature_tests/upgrade/controlplane_upgrade/3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh (hunk 1 above)
Comment (smoshiur1237) [reply scripts/feature_tests/upgrade/controlplane_upgrade/3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh:60]: Done

**Comment by jan-est** [comment]
lgtm

**Pull request merged**